apps/src/applab/applab.js

```diff
@@ -1332,10 +1335,17 @@ Applab.onCodeModeButton = function() {
 /**
  * If the filename is relative (contains no slashes), then prepend
  * the path to the assets directory for this project to the filename.
+ *
+ * If the filename is external and non-https, route it through the MEDIA_PROXY.
```

Comment:
I wasn't sure what you mean by "external and non-https";  the pattern below only checks for http:// rather than "internal vs. external check".  I